src/main/java/com/sequenceiq/cloudbreak/service/stack/connector/aws/SnsTopicManager.java

```diff
@@ -1,141 +0,0 @@
-package com.sequenceiq.cloudbreak.service.stack.connector.aws;
-
-import java.util.List;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.stereotype.Service;
-
-import com.amazonaws.regions.Regions;
-import com.amazonaws.services.sns.AmazonSNSClient;
-import com.amazonaws.services.sns.model.ConfirmSubscriptionResult;
-import com.amazonaws.services.sns.model.CreateTopicResult;
-import com.google.common.annotations.VisibleForTesting;
-import com.sequenceiq.cloudbreak.conf.ReactorConfig;
-import com.sequenceiq.cloudbreak.domain.AwsCredential;
-import com.sequenceiq.cloudbreak.domain.CloudPlatform;
-import com.sequenceiq.cloudbreak.domain.SnsRequest;
-import com.sequenceiq.cloudbreak.domain.SnsTopic;
-import com.sequenceiq.cloudbreak.domain.Stack;
-import com.sequenceiq.cloudbreak.logger.MDCBuilder;
-import com.sequenceiq.cloudbreak.repository.SnsTopicRepository;
-import com.sequenceiq.cloudbreak.repository.StackRepository;
-import com.sequenceiq.cloudbreak.service.credential.aws.CrossAccountCredentialsProvider;
-import com.sequenceiq.cloudbreak.service.stack.event.ProvisionSetupComplete;
-
-import reactor.core.Reactor;
-import reactor.event.Event;
-
-@Service
-public class SnsTopicManager {
-
-    public static final String NOTIFICATION_TOPIC_ARN_KEY = "notificationTopicArn";
-
-    private static final String CB_TOPIC_NAME = "cloudbreak-notifications";
-
-    private static final Logger LOGGER = LoggerFactory.getLogger(SnsTopicManager.class);
-
-    @Value("${cb.host.addr}")
-    private String hostAddress;
-
-    @Value("${cb.sns.ssl}")
```

Comment:
please make sure to remove this environment variable from the deployment scripts before merge